packages/rancher-istio/1.15/rancher-istio/charts/configs/istio-base.yaml

```diff
@@ -61,15 +61,6 @@ spec:
             targetPort: 15443
     istiodRemote:
       enabled: {{ .Values.istiodRemote.enabled }}
-      k8s:
-        nodeSelector: {{ include "linux-node-selector" . | nindent 12 }}
-{{- if .Values.nodeSelector }}
-{{- toYaml .Values.nodeSelector | nindent 12 }}
-{{- end }}
-        tolerations: {{ include "linux-node-tolerations" . | nindent 12 }}
-{{- if .Values.tolerations }}
-{{- toYaml .Values.tolerations | nindent 12 }}
-{{- end }}
```

Comment:
Can you elaborate a bit more on why this was removed? We use the "linux-node-*" node selectors and tolerations together with the permits-os annotation to deploy charts in hybrid clusters (linux and windows nodes). Could removing this prevent pods from being scheduled in the correct node? I'm not familiar with istio, so it might be good to have an additional reviewer.

Reply:
I have done make forward port for [100.5.0+up1.15.3](https://github.com/rancher/charts/tree/dev-v2.6/charts/rancher-istio/100.5.0%2Bup1.15.3)

istiod remote component works in the same fashion as base. These tolerations were added in version [100.1.2+up1.11.7](https://github.com/rancher/charts/tree/dev-v2.6/charts/rancher-istio/100.1.2%2Bup1.11.7) and since then https://github.com/rancher/rancher/issues/39680 this issue occurs. That's why to solve it i have removed these tolerations.

ref: istio operator component maps: https://github.com/istio/istio/blob/1aca7a67afd7b3e1d24fafb2fbfbeaf1e41534c0/operator/pkg/translate/translate.go#L120
here there is no resource associated with istiodRemote. which is the same case for base component.
earlier tolerations for base component were also removed by the PR https://github.com/rancher/charts/pull/1817